src/view/com/util/forms/PostDropdownBtnMenuItems.tsx

```diff
@@ -691,6 +712,16 @@ let PostDropdownMenuItems = ({
         confirmButtonCta={_(msg`Share anyway`)}
       />
 
+      <Prompt.Basic
+        control={copyPostUrlLoggedOutWarningPromptControl}
+        title={_(msg`Note about sharing`)}
+        description={_(
+          msg`This post is only visible to logged-in users. It won't be visible to people who aren't logged in.`,
+        )}
+        onConfirm={onCopyPostUrl}
+        confirmButtonCta={_(msg`Share anyway`)}
+      />
```

Comment:
I duplicated the prompt for simplicity, but we might want to change the copy here. Copying a link might not always result in sharing, but it's most likely the case.